org.eclipse.scanning.points/src/org/eclipse/scanning/points/PointGeneratorService.java

```diff
@@ -57,17 +58,15 @@
 
 public class PointGeneratorService implements IPointGeneratorService {
 
-	@SuppressWarnings("rawtypes")
-	private static final Map<Class<? extends IScanPathModel>, Class<? extends IPointGenerator>> generators;
+	private static final Map<Class<? extends IScanPathModel>, Class<? extends IPointGenerator<?>>> generators;
 	private static final Map<String,   GeneratorInfo>                                           info;
 
 	// Use a factory pattern to register the types.
 	// This pattern can always be replaced by extension points
 	// to allow point generators to be dynamically registered.
 	static {
 		System.out.println("Starting generator service");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)